src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.csproj

```diff
@@ -51,19 +66,33 @@
     <Folder Include="Cmdlets\Restore\" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\Helpers\Helpers.csproj">
+    <ProjectReference Include="..\..\Common\Commands.Common\Commands.Common.csproj">
```

Comment:
all dll names should be fqdn ex: Microsoft.azure.recoveryservices.backup.cmdlets.dll, Microsoft.azure.recoveryservices.backup.providers.dll,  : Microsoft.azure.recoveryservices.backup.models.dll etc